Orange/tests/test_variable.py

```diff
@@ -99,16 +121,23 @@ def test_adjust_decimals(self):
         a.val_from_str_add("5.1234")
         self.assertEqual(a.str_val(4.654321), "4.6543")
 
+    def dont_pickle_anonymous_variables(self):
```

Comment:
Which one?

If you try to pickle variables without names, it raises an exception.

If you refer to test_adjust_decimals, it val_from_str_add adjusts the number of decimals to the largest number it sees in the string data. Since the `add` was given four decimals, `str_val` should give four decimals.